src/main/configService.ts

```diff
@@ -167,13 +57,19 @@ export default class ConfigService extends EventEmitter {
 
             if (fn === 'get') {
                 const value = this.get(key);
-                console.log('[Config Service] Get from config store:', key, value);
                 event.returnValue = value;
             } else
             if (fn === 'set') {
                 const value = args[2];
                 this.set(key, value);
-                console.log('[Config Service] Set in config store:', key, value);
+
+                this._changedProperties[key] = value;
```

Comment:
Why do we need this variable? I don't see any uses of it. It feels a bit hacky to have this tracked as such, I don't have any objection to the actual code here but if we're going to have something checking this we're opening up opportunity for timing problems. 

If we really need this, then I'd prefer a pubsub style mechanism that we store the "changed" property on some object, like this, and then dismiss it once we've processed it.

Reply:
I'll find a different way to achieve what I intended. The intended result was that we only log the things that have changed/been set - this could be achieved in the settings window instead, collecting the values that have changed and saving those in bulk instead. I'll work on that.

I agree this solution wasn't the prettiest.

Reply:
Better solution was found - it now has a `set_values` function that checks the object being passed against the current configuration before setting anything, then logs the changes afterwards.